ansible_base/rbac/management/commands/RBAC_checks.py

```diff
@@ -0,0 +1,50 @@
+from collections import defaultdict
+
+from django.core.management.base import BaseCommand
+
+from ansible_base.rbac import permission_registry
+from ansible_base.rbac.models import ObjectRole, RoleDefinition
+
+
+class Command(BaseCommand):
+    help = "Runs bug checking sanity checks, gets scale metrics, and reccomendations for Role Based Access Control"
+
+    def handle(self, *args, **options):
+        rd_ct = RoleDefinition.objects.count()
+        self.stdout.write(f'Inspecting {rd_ct} role definitions')
+        self.stdout.write('  checking for minimum of view permission')
+        indexed_rds = defaultdict(list)
+        for rd in RoleDefinition.objects.prefetch_related('permissions'):
+            perm_list = list(rd.permissions.values_list('codename', flat=True))
+            if 'view' not in ' '.join(perm_list):
```

Comment:
This seems like not the best idea.  Perhaps `if not any(p.startswith('view_') for p in perm_list):` (or whatever the pattern is for these permission tokens)?

We just had the bug in AWX with substring matching on exceptions, and I know that has to do with language localization which will not be operative here, but this also seems like a bug waiting to happen due to its fuzziness.